src/global/interfaces/accessibility-properties.ts

```diff
@@ -1,21 +0,0 @@
-/**
- * The accessibility interface is used to enforce all accessibility labels to be implemented at once.
- */
-export interface AccessibilityProperties {
```

Comment:
I'd keep the interface for the remaining usages, so we can still maintain a consistent naming.